Library/Homebrew/test/dev-cmd/audit_spec.rb

```diff
@@ -854,18 +854,12 @@ class Foo < Formula
     end
 
     describe "#audit_revision_and_version_scheme" do
-      subject {
-        fa = described_class.new(Formulary.factory(formula_path), git: true)
-        fa.audit_revision_and_version_scheme
-        fa.problems.first&.fetch(:message)
-      }
-
-      let(:origin_tap_path) { Tap::TAP_DIRECTORY/"homebrew/homebrew-foo" }
-      let(:foo_version) { Count.increment }
-      let(:formula_subpath) { "Formula/foo#{foo_version}.rb" }
-      let(:origin_formula_path) { origin_tap_path/formula_subpath }
-      let(:tap_path) { Tap::TAP_DIRECTORY/"homebrew/homebrew-bar" }
       let(:formula_path) { tap_path/formula_subpath }
+      let(:tap_path) { Tap::TAP_DIRECTORY/"homebrew/homebrew-bar" }
+      let(:origin_formula_path) { origin_tap_path/formula_subpath }
+      let(:formula_subpath) { "Formula/foo#{foo_version}.rb" }
+      let(:foo_version) { Count.increment }
+      let(:origin_tap_path) { Tap::TAP_DIRECTORY/"homebrew/homebrew-foo" }
```

Comment:
same here, please don't reorder these, thanks!

Reply:
The linter did this stuff

Reply:
The linter moved `tap_path` from the bottom of this list to the top? Which linter?

Reply:
Whichever one did the fixups. Sorry. I need to go to a funeral.

Reply:
I actually _can_ reproduce this by pasting the new test in before the `subject` on line 859. Then if you run `brew style --fix` it tries to reorder it so the `let`s and `begin` are before the test and somehow that moves the `subject` and reorders the lines. I think the reason is that it tries to move each `let` individually to be above the new example group instead of moving all of the `let`s together, effectively reversing their order. Same with `begin`. It must not check the location of `subject`.

In general, if `brew style --fix` makes big changes like this for seemingly no reason, it's probably a sign that it didn't quite do what you want. You should be able to look at the output of `brew style` to figure out what it's complaining about and make those changes manually in those cases.

Reply:
Sure. I ran into a case where `clang-format` reflowed something and it was clear that wasn't the goal.

I will not have any time to try to figure this out before next week.

I barely speak Ruby.

Anyway. I'm off to the funeral. Have a good weekend.